Doc/whatsnew/3.10.rst

```diff
@@ -2199,7 +2197,7 @@ Porting to Python 3.10
   ``picklebufobject.h``, ``pyarena.h``, ``pyctype.h``, ``pydebug.h``,
   ``pyfpe.h``, and ``pytime.h`` have been moved to the ``Include/cpython``
   directory. These files must not be included directly, as they are already
-  included in ``Python.h``: :ref:`Include Files <api-includes>`. If they have
+  included in ``Python.h``: :ref:`api-includes`. If they have
```

Comment:
```suggestion
  included in ``Python.h``; see :ref:`api-includes`. If they have
```

The current syntax doesn't really make grammatical sense, so I suggest fixing it.